README.md

```diff
@@ -18,6 +18,10 @@ Throughout the repository READMEs, the prompt `~/repo>` is used to denote a `bas
 
 Refer to our [quickstart guide](./QUICKSTART.md) to learn about environment setup and common tasks without further ado.
 
+## Supported platforms
+
+Unless otherwise stated, only x86 platforms are supported for building this project and running the tools provided. It is possible, however, to build and run the [TCPSigner bundle](./utils/tcpsigner-bundle/README.md) on arm64 platforms.
```

Comment:
```suggestion
Unless otherwise stated, only x86 platforms are supported for building this project and running the tools provided. It is possible, however, to build and run the [TCPSigner bundle](./utils/tcpsigner-bundle/README.md) on arm64 platforms. This is provided for development and testing purposes.
```